global_planner/src/planner_core.cpp

```diff
@@ -221,11 +223,24 @@ bool GlobalPlanner::makePlan(const geometry_msgs::PoseStamped& start, const geom
 
 bool GlobalPlanner::makePlan(const geometry_msgs::PoseStamped& start, const geometry_msgs::PoseStamped& goal,
                            double tolerance, std::vector<geometry_msgs::PoseStamped>& plan) {
+    double cost; // Dummy
+    std::string message; //Dummy
+    return makePlan(start, goal, tolerance, plan, cost, message) == mbf_msgs::GetPathResult::SUCCESS;
+}
+
+uint32_t GlobalPlanner::makePlan(const geometry_msgs::PoseStamped &start, const geometry_msgs::PoseStamped &goal,
+                  double tolerance, std::vector<geometry_msgs::PoseStamped> &plan, double &cost, std::string &/*message*/) {
```

Comment:
- /*message*/ ? 
- where is the cost computed
- where is the message set? -> The message should be defined following to the error meaning.